pkg/collector/corechecks/snmp/common/contants.go

```diff
@@ -2,3 +2,6 @@ package common
 
 // MetadataDeviceResource is the device resource name
 const MetadataDeviceResource = "device"
+
+// SnmpIntegrationName is the name of the snmp integration
```

Comment:
Would be nice if we can make it explicit what "name" refers to. It's used as an id in the backend for external tags, right?

Reply:
> Would be nice if we can make it explicit what "name" refers to. 

Sure, I can make it more explicit.

> It's used as an id in the backend for external tags, right?

We are using `snmp` as source type for external tags.

My understanding is that the source type is an information that is stored in HMS. For example in Infra List we can see an SNMP section:
![image](https://user-images.githubusercontent.com/49917914/144867085-6da36ea8-32e5-4168-99a6-ca57b7a66834.png)

But in NDM case, agent tags and external tags are merged together (which I think is fine for now):

![image](https://user-images.githubusercontent.com/49917914/144867317-ca63d9ef-9906-45d5-b4f2-bd859550d423.png)

cc @pducolin

Reply:
Do you need a new entry ? Currently we get tags once from enriched tags from EP intake enrichment step. Metadata have all tags, as they are required for filtering.

We then split in user tags / agent tags by getting tags from HMS and checking the `Users` source https://github.com/DataDog/web-ui/blob/07af12757768cf930b67306b9a01b98c846c539e/javascript/datadog/infrastructure/lib/devices/hooks/use-custom-tags.ts#L47

We would need to add a new entry here to parse also snmp tags. But, is `SNMP` clear as a name for external tags ? If I understood it correctly, external tags are those defined in the agent root config, and `Datadog agent` tags those defined in snmp config. Using `snmp` for the first might be confusing